InputDate/InputDateNavBar/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DayPicker from 'react-day-picker';
+import Select from '../../Select';
+import { ArrowRightIcon, ArrowLeftIcon } from '../../Icon/Icons';
+import Button from '../../Button';
+import moment from 'moment';
+
+const NavBar = ({
+  month,
+  nextMonth,
+  previousMonth,
+  onPreviousClick,
+  onNextClick,
+  className,
+  localeUtils,
+  firstMonthToDisplay,
+  onNavigationClick,
+}) => {
+  const styles = {
+    navbar: {
+      position: 'absolute',
+      left: 'unset',
+      right: 0,
+      width: '2.5rem',
+    },
+    left: {
+      float: 'left',
+    },
+    right: {
+      float: 'right',
+    },
+    align: {
+      verticalAlign: 'middle',
+    },
+  };
+
+  const onNext = () => {
```

Comment:
This could be moved outside the render function (with `nextMonth`) as the argument and would make it so the function is constructed only once :smile

Reply:
I would still leave it inside as they depend not only on the `nextMonth` and `previousMonth` but also the `onPreviousClick` and `onNextClick` props, all coming from the NavBar.